modules/core/src/models/config.ts

```diff
@@ -238,6 +238,10 @@ export interface CommonUIConfigInterface<T = Record<string, unknown>> {
      * Theme configs.
      */
     theme: AppThemeConfigInterface;
+    /**
+     * Config to check whether consent is required for trusted apps.
+     */
+    isTrustedAppConsentRequired?: boolean;
```

Comment:
I think we don't need to add this here since it's already included in the admin.core.v1 models